tiledb/sm/storage_manager/storage_manager.cc

```diff
@@ -222,26 +222,26 @@ StorageManager::load_array_schemas_and_fragment_metadata(
   const auto& fragments_to_load = filtered_fragment_uris.fragment_uris();
 
   // Get the consolidated fragment metadatas
-  std::vector<Buffer> f_buffs(meta_uris.size());
+  std::vector<Tile> f_tiles(meta_uris.size());
```

Comment:
Could we use a std::vector<shared_ptr<Tile>> here to prevent calling the Tile() constructor? That will be helpful later for C.41 work.

Reply:
changed to std::vector<shared_ptr<Tile>>.